internal/distro/rhel90/package_sets.go

```diff
@@ -668,6 +668,7 @@ func edgeSimplifiedInstallerPackageSet(t *imageType) rpmmd.PackageSet {
 			"policycoreutils",
 			"policycoreutils-python-utils",
 			"procps-ng",
+			"rdma-core",
```

Comment:
this can be dropped now that #2045 merged

Reply:
Done.  Thanks!